qlib/pagetable_cc.rs

```diff
@@ -0,0 +1,1447 @@
+pub use super::linux_def::*;
```

Comment:
would it be better to have a cc directory, instead of having files suffixed "cc"? There should be more such files I suppose? (like `Kernel_cc.rs`, and maybe there are more to come in the future).

Reply:
Yes, I can move `Kernel_cc.rs` and `pagetable_cc.rs` into the cc folder which already exists. But I am not sure that I should put all the cc related codes into that folder, or it might be easy to add support for cc if you put them together in one folder. You can easily realize there is a related cc file without opening the cc folder.

Reply:
Then how to differ from the normal ones? For example, they are both functions of Pagetable struct, and I need to decide which to run in the runtime.